test/y2partitioner/actions/controllers/filesystem_test.rb

```diff
@@ -595,13 +595,6 @@ def create_partition(size)
         expect(subject.partition_id).to eq(type.default_partition_id)
       end
 
-      context "when the previous filesystem exists in the disk" do
```

Comment:
For the same reason than above, I would not simply delete this test, but I would state here than the label from the previous file-system is reused on purpose.